tutorial/README.md

```diff
@@ -0,0 +1,190 @@
+
+Esky Tutorial
+=============
+
+This directory contains a brief tutorial on using esky in your frozen 
+applications. You will need a freezer module installed.
+
+This guide will focus on telling you what to do and to get your feet wet,
+jump into the source code of the examples for the details.
+
+## Choosing a Freezer
+
+Esky currently supports
+ - cx_freeze
+ - py2exe, 
+ - py2app 
+ - bbfreeze
+
+We suggest that you use **cx_freeze** because it is/supports:
+ - python2
+ - python3
+ - maintained 
+ - linux
+ - mac 
+ - windows
+
+Esky is currently not working with py2exe, patches are welcome!,
+
+Bbfreeze is unmaintained!
```

Comment:
Perhaps we should just tip it out then, there's no good reason to keep maintaining our support here.  Can you please add more words here like "...and support will likely be removed in a future release of esky"?